packages/-ember-data/tests/unit/model/relationships/belongs-to-test.js

```diff
@@ -120,50 +114,46 @@ module('unit/model/relationships - DS.belongsTo', function(hooks) {
 
     adapter.shouldBackgroundReloadRecord = () => false;
 
-    run(() => {
-      store.push({
-        data: [
-          {
-            type: 'tag',
-            id: 1,
-            attributes: {
-              name: 'whatever',
-            },
+    store.push({
```

Comment:
You don't seem to wait in this test, and we were using the runloop to make sure the observers would trigger on time

Reply:
Do you think that since the test passes, we can do without `await`?

Reply:
No, because it might be passing due to timing, and depending on the ordering/timing of the test runs works out, might not pass in the future